pkg/k8shandler/podtemplate.go

```diff
@@ -0,0 +1,90 @@
+package k8shandler
```

Comment:
Would this not more naturally fit into https://github.com/openshift/elasticsearch-operator/tree/master/pkg/utils/comparators ?

Reply:
i think when i was doing that before it ended up creating a cyclic dependency... but that may no longer be the case